Content.Server/Radio/EntitySystems/RadioDeviceSystem.cs

```diff
@@ -216,9 +216,17 @@ private void OnReceiveRadio(EntityUid uid, RadioSpeakerComponent component, ref
         var name = Loc.GetString("speech-name-relay", ("speaker", Name(uid)),
             ("originalName", nameEv.Name));
 
-        // log to chat so people can identity the speaker/source, but avoid clogging ghost chat if there are many radios
-        var message = args.OriginalChatMsg.Message; // The chat system will handle the rest and re-obfuscate if needed.
-        _chat.TrySendInGameICMessage(uid, message, InGameICChatType.Whisper, ChatTransmitRange.GhostRangeLimit, nameOverride: name, checkRadioPrefix: false, languageOverride: args.Language);
+        {
```

Comment:
Why is this in a new scoped block?

Reply:
I had set the files aside when I moved to the new PR branch, must've copied it wrong, I'll push the proper code